dynawo/sources/Common/Dictionaries/DYNLog_en_GB.dic

```diff
@@ -100,6 +100,7 @@ HvdcExtDynModel               =             HVDC %1% using an external dynamic m
 AddingHvdcToNetwork           =             adding HVDC %1% to the Network.
 TapChangerLocked              =             %1%:  Tap changer is blocked
 NetworkInitSwitchCurrentsFailed =           model network : initialization of switches' currents failed
+NetworkInitInternalVarFailed  =             model network : initialization of internal variables from dump failed
```

Comment:
As the notion of internal variables is really embedded into Dynawo (it's a kind of a hack to avoid declaring them as variables) I'm not sure we need to create a new message, maybe the same error message as the load of variables could be used?

Reply:
Agree. New log was removed.